Could not resolve "https"

<!-- This is adapted from github.com/imba/imba -->

### formidablejs/framework version: 0.13.0

<!-- Provide the exact version of formidablejs/framework in which you see the bug.  -->

### formidablejs/view version: 0.2.5

<!-- Provide the exact version of formidablejs/view in which you see the bug.  -->

### Details
Getting:
The package "https" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.

<!--
  Provide a more detailed introduction to the issue itself, and why you consider it to be bug.
  How has this bug affected you?
  What were you trying to accomplish?
-->

### Expected Behavior
It should just work

<!-- Tell us what should happen -->

### Actual Behavior
Application is not starting due to this bug
<!-- Tell us what happens instead -->

### Possible Fix
Not sure yet
<!-- Not obligatory, but suggest a fix or reason for the bug -->

<details><summary>Additional Info</summary>

### Your Environment

<!-- Include as many relevant details about the environment you experienced the bug in -->

- Node 16.15
- Mac 12.6

### Steps to Reproduce

<!-- or an unambiguous set of steps to reproduce this bug -->

<!-- include code to reproduce, if relevant -->

Run the following command:
`formidable new <app-name> --type "full-stack" --stack "imba" --scaffolding "spa" --ts`

### Stack Trace

<!-- If an error is thrown, provide the stack trace here -->

```
✘ [ERROR] Could not resolve "https"

    node_modules/axios/dist/node/axios.cjs:8:22:
      8 │ const https = require('https');
        ╵                       ~~~~~~~

  The package "https" wasn't found on the file system but is built into node. Are you trying to
  bundle for node? You can use "platform: 'node'" to do that, which will remove this error.

✘ [ERROR] Could not resolve "https"

    node_modules/follow-redirects/index.js:4:20:
      4 │ var https = require("https");
        │                     ~~~~~~~
        ╵                     "./https"

  The package "https" wasn't found on the file system but is built into node. Are you trying to
  bundle for node? You can use "platform: 'node'" to do that, which will remove this error.

× failed with 2 errors
```

</details>

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@formidablejs/view",
-  "version": "0.2.5",
+  "version": "0.2.6",
   "description": "Building blocks for Formidable SPA's",
   "author": "Donald Pakkies",
   "license": "MIT",

